modules/030-cloud-provider-aws/docs/USAGE.md

```diff
@@ -2,7 +2,9 @@
 title: "Cloud provider — AWS: usage"
 ---
 
-## An example of the `AWSInstanceClass` CR
+## Custom resource `AWSInstanceClass`
```

Comment:
```suggestion
## An example of the `AWSInstanceClass` custom resourse
```
I think that the first mention of something on the page should still be indicated in full. @z9r5  What do you think?